source/extensions/matching/input_matchers/ip/matcher.cc

```diff
@@ -0,0 +1,33 @@
+#include "extensions/matching/input_matchers/ip/matcher.h"
+
+#include "common/network/utility.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Matching {
+namespace InputMatchers {
+namespace IP {
+
+Matcher::Matcher(std::vector<Network::Address::CidrRange>&& ranges)
+    : trie_({{true, std::move(ranges)}}) {}
```

Comment:
Is true vs false here important? Maybe add a comment?

Reply:
Comment added in https://github.com/envoyproxy/envoy/pull/16592/commits/a0360a73c6b3df1e3a75191f813939fe8ce49608